webapp/channels/src/sass/components/_modal.scss

```diff
@@ -242,77 +240,62 @@
 
     .modal-header {
         display: flex;
-        min-height: 56px;
-        padding: 14px 48px 11px 15px;
-        border-bottom: 1px solid $light-gray;
-        background: #333;
-        border-radius: 0;
-        border-radius: 3px 3px 0 0;
-        color: $white;
-
-        @include clearfix;
-
-        .modal-title {
-            overflow: hidden;
-            max-width: calc(100% - 80px);
-            flex: 1;
-            color: var(--sidebar-header-text-color);
-            font-size: 17px;
-            line-height: 27px;
-            text-overflow: ellipsis;
-            white-space: nowrap;
-
-            .name {
-                font-weight: 600;
-            }
-
-            .shared-channel-icon {
-                width: 16px;
-                height: 16px;
-                margin: 0 8px;
-                color: #3d3c40;
-                font-size: 16px;
-                line-height: 16px;
+        align-items: center;
+        justify-content: space-between;
+        padding: 24px 64px 24px 32px;
+        border: none;
+        background: transparent;
 
-                &::before {
-                    margin: 0;
-                }
-            }
+        &.divider {
+            border-bottom: 1px solid rgba(var(--center-channel-color-rgb), 0.12);
         }
 
-        .modal-action {
-            padding: 0;
-            margin: 0;
+        &::before,
+        &::after {
+            content: none;
         }
 
         .close {
             position: absolute;
-            z-index: 5;
-            right: 10px;
-            width: 30px;
-            height: 30px;
-            color: white;
-            font-size: 24px;
-            line-height: 30px;
-            opacity: 0.7;
-            text-shadow: none;
-            transition: all 0.1s ease-in;
+            top: 22px;
```

Comment:
Can we change this to `top: 18px;`? Seems like it needs to shift up a bit to align better with the title

Reply:
@matthewbirtch I don't that would work. I think previously it was 18px, but that doesn't work when we add a button to the mix.

You can try it with the user groups modal.

Reply:
This is what it looks like with top 18px.
<img width="746" alt="Screenshot 2023-12-15 at 2 56 28 PM" src="https://github.com/mattermost/mattermost/assets/11034289/ac59add8-f9ef-4751-a4db-0b3c486a63a2">